queue_services/business-events-listener/src/business_events_listener/worker.py

```diff
@@ -134,7 +134,8 @@ async def process_name_events(event_message: Dict[str, any]):
                 affiliation: AffiliationModel = AffiliationModel(entity=nr_entity, org=org)
                 affiliation.flush()
                 activity: ActivityModel = ActivityModel(org_id=org.id, action=ActivityAction.CREATE_AFFILIATION.value,
-                                                        item_name=nr_entity.name, item_id=nr_entity.business_identifier,
```

Comment:
There is no name at this point only the identifier.. as the user can change the name until it's staff reviewed.